create_heatmaps.py

```diff
@@ -21,6 +21,7 @@
 
 parser = argparse.ArgumentParser(description='create heatmaps from lightcurve data')
 parser.add_argument('--config_path', type=str, help='absolute or relative path to your yml config file, i.e. "/user/files/create_heatmaps_config.yml"')
+parser.add_argument('--index', type=int, help='index of single lc file you wish to create heatmaps for within your config file, i.e. 4. if no index is provided, heatmaps will be created for objects in all lc files')
```

Comment:
I think we agreed to keep your copy of `create_heatmaps` separate from the repository, so could you revert your changes to this file? 

`git checkout -- create_heatmaps.py`

Reply:
should `Updated 0 paths from the index` be the output when I run that line?